packages/wallet-protocols/src/protocols/depositing/protocol.ts

```diff
@@ -1,10 +1,11 @@
 import { bigNumberify } from 'ethers/utils';
```

Comment:
The `depositing` protocol has been migrated to `workflow\depositing`, I probably should have flagged this one as obsolete :(